src/vision.cs

```diff
@@ -2054,6 +2342,21 @@ interface VNRequest : NSCopying {
 		//[Static]
 		//[Export ("currentRevision")]
 		//VNRequestRevision CurrentRevision { get; }
+
+		[TV(17, 0), Mac(14, 0), iOS(17, 0), MacCatalyst(17, 0)]
+		[Export("supportedComputeStageDevicesAndReturnError:")]
+		[return: NullAllowed]
+		NSDictionary<NSString, NSArray<IMLComputeDeviceProtocol>> GetSupportedComputeDevices([NullAllowed] out NSError error);
+
+		// could this be combined into a Get/Set pair? docs are a bit weird..
```

Comment:
thoughts?

Reply:
You would need to use an indexer, since it takes the computeStage: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/indexers/

And indexers do not have names, so you won't be able to diff with another indexer that takes a string. Leaving it in two diff methods is a better approach.